pandas-stubs/core/series.pyi

```diff
@@ -1598,7 +1598,7 @@ class Series(IndexOpsMixin[S1], NDFrame):
         bins: int | None = ...,
         dropna: _bool = ...,
     ) -> Series[float]: ...
-    def transpose(self, *args, **kwargs) -> Series[S1]: ...
+    def transpose(self, axes: None = ...) -> Series[S1]: ...
```

Comment:
The only argument that pandas actually lets us pass at runtime is `axes=None`

https://github.com/pandas-dev/pandas/blob/d831326e11971af61307e4fc940855556ef3cdb7/pandas/compat/numpy/function.py#L317-L320

For Series and Index, it can be passed positionally or by name

For DataFrame, it can only be passed positionally